draft-documents/satisfying-real-world-use-cases.md

```diff
@@ -436,7 +436,7 @@ Thus, the main technical needs for Marsha's case are:
 
 The key to meeting these needs is to provide an individual services across time
 and space, in a way that avoids duplication and provides for continuity of care
-(medically)--in other words, persistent correlatable claims.  An independent
+(medically)--in other words, persistent correlatable identifier for a particular context - along with associated records.  An independent
```

Comment:
How about "a persistent correlatable identifier valid in the appropriately limited context of its associated records, and nowhere else"?